src/main/java/nl/knaw/dans/vaultingest/client/AbstractDepositValidator.java

```diff
@@ -63,6 +65,10 @@ public void validate(Path depositDir) throws InvalidDepositException {
                 }
             }
         }
+        catch (ProcessingException e) {
+            log.error("Unable to connect to validator service on URL {}", serviceUri);
```

Comment:
Looks to me this log message is jumping to conclusions.

Reply:
Why is that?

Reply:
When I don't see an explicit connection exception, I suppose there can be other causes too.